### DISCUSSION TITLE
JDK8 RestEasy-Client Version

### DISCUSSION THREAD

**Discussion by blacklabelops-legacy** [resteasy/resteasy#3507] [Q&A] [answered]
Hello,

Can you please tell me which RestEasy-Client-Major-Version to use in a JDK 8 project?

Thanks!

**Comment by jamezp** [comment, marked as answer]
Hello @blacklabelops-legacy,
RESTEasy 5.0.6.Final is the latest version that runs on Java 8. RESTEasy 6.2.3.Final requires Java 11 as the Jakarta REST 3.1 specification requires Java 11.
